local/php/toolbar.go

```diff
@@ -116,19 +116,22 @@ func (p *Server) tweakToolbar(body io.ReadCloser, env map[string]string) (io.Rea
 		return body, errors.WithStack(err)
 	}
 	content := []byte(`
-<div class="sf-cli sf-toolbar-block sf-toolbar-block-sf-cli ` + logoBg + ` sf-toolbar-block-right">
+<div class="sf-cli sf-toolbar-block-config sf-toolbar-block sf-toolbar-block-sf-cli ` + logoBg + ` sf-toolbar-block-right">
```

Comment:
I'm quite sure `sf-toolbar-block-config` is expected to be used only by the config panel.

Reply:
I'm quite sure the bad display in small screen is a bug in WebprofilerBundle where one of the CSS selector uses the `sf-cli` class instead of `sf-toolbar-block-sf-cli`

Reply:
> I'm quite sure `sf-toolbar-block-config` is expected to be used only by the config panel.

that's what I thought first. But then I came to the conslusion that is this a class instead of an id that would probably be supported 🙂

> I'm quite sure the bad display in small screen is a bug in WebprofilerBundle where one of the CSS selector uses the sf-cli class instead of sf-toolbar-block-sf-cli

TBH I didn't expect the WebProfilerBundle to include any specific CSS for the CLI so I was surprised to see `sf-toolbar-block-sf-cli` in there 😮 
Then I guess I will open a PR on symfony/symfony ;)

Reply:
NVM it has been fixed in https://github.com/symfony/symfony/pull/48961

Reply:
reverted